examples/version-migration/separate-container/src/start.ts

```diff
@@ -4,15 +4,19 @@
  */
 
 import type {
-	IMigratableModel,
-	IMigrationTool,
-	IVersionedModel,
+	IMigrator,
+	IMigratorEntryPoint,
+	ImportDataCallback,
 } from "@fluid-example/migration-tools/internal";
 import {
-	getModelAndMigrationToolFromContainer,
-	Migrator,
-	SimpleLoader,
+	makeCreateDetachedContainerCallback,
+	makeSeparateContainerMigrationCallback,
 } from "@fluid-example/migration-tools/internal";
+import type { IContainer } from "@fluidframework/container-definitions/internal";
+import {
+	type ILoaderProps,
+	loadExistingContainer,
+} from "@fluidframework/container-loader/internal";
 import { RouterliciousDocumentServiceFactory } from "@fluidframework/routerlicious-driver/internal";
```

Comment:
Can we avoid /internal imports in here? (Examples should not use /internal APIs.)
Perhaps update .eslintrc.cjs to have:
```json
		"import/no-internal-modules": [
			"error",
			{
				allow: [
					// Within Fluid Framework allow import of '/alpha' from other FF packages.
					"@fluidframework/*/alpha",
```
or similar

Reply:
Hmm, in general our in-repo examples do use /internal - I can give it a shot for some of these though

Reply:
The problem seems to be that I need the full exception list to concatenate (otherwise this simply overrides the existing exception list) and everything blows up.  I could duplicate that exception list here but that has its own drawbacks.

Might be good to consider a centralized override for examples I suppose